Sources/MongoSwift/ReadPreference.swift

```diff
@@ -0,0 +1,164 @@
+import libmongoc
+
+/// A class to represent a MongoDB read preference.
+final public class ReadPreference {
+
+    /// An enumeration of possible ReadPreference modes.
+    public enum Mode: String {
+        /// See https://docs.mongodb.com/manual/reference/read-preference/
```

Comment:
let's move this comment up one line. with its current position, it will only show up in the docs under the `.primary` case, but it should be associated with the `Mode` type as a whole. or we could even put it on the `ReadPreference` class itself I suppose. 

you could also list it as a "see also", like [here](https://github.com/mongodb/mongo-swift-driver/blob/c3e9c02711fcab5a88d39156ecb1df2cf1575e3f/Sources/MongoSwift/MongoClient.swift#L106), which then shows up nicely in the [docs](https://mongodb.github.io/mongo-swift-driver/Classes/MongoClient.html#/s:10MongoSwift0A6ClientCACSS16connectionString_AA0C7OptionsVSg7optionstKcfc). 
though given that we have no info on the modes here, maybe the "also" part doesn't make sense... I don't have strong feelings beyond moving it from this spot to somewhere more useful

Reply:
I think a "SeeAlso" at the class level makes the most sense. Will do.